dist/stat_data.py

```diff
@@ -335,6 +339,41 @@ def __init__(self, name, desc, flags=''):
     CapacityStat('fsync_all_fh_total', 'background fsync file handles considered'),
     CapacityStat('fsync_all_time', 'background fsync time (msecs)', 'no_clear,no_scale'),
 
+    ##########################################
+    # Checkpoint statistics
+    ##########################################
+    CheckpointStat('checkpoints', 'transaction checkpoints'),
+    CheckpointStat('checkpoint_custom_dsrc', 'number of custom data sources checkpointed'),
```

Comment:
Do we care about this? We have no other stats that apply to custom data sources and MongoDB doesn't use them. So this stat will just take up space in the stat structure.

Reply:
My logic here was that it's a potentially expensive black box that is otherwise totally invisible in the stats. If the use case isn't there I'll remove it - I understand Mongo doesn't use it, would it be worthwhile leaving it for any other users?

Reply:
I think we should only add them if/when there is a need. I wouldn't be surprised if custom data sources is on the list of unused things to consider for removal.

Reply:
Ack, removed in 1f832af. Thanks for being patient with this.